doc/flame/components.md

```diff
@@ -419,19 +419,50 @@ use `animation.completed`.
 Example:
 
 ```dart
-await animation.completed;
+await
+animation.completed;
```

Comment:
Why are these on two lines? :thinking:

Reply:
I don't know, thats how it was when I pulled it and I just left it be.

Reply:
It's not like that on `main` so you probably accidentally put a new line there.

Reply:
Nah, its the darn auto format.  I just fixed it and it immediately put it back on two lines.  I had to disable the auto-formatter.  Maybe its out of date.